docs/analyzer-rules/AZFW0011.md

```diff
@@ -0,0 +1,32 @@
+# AZFW0011: Invalid binding type
+
+| | Value |
+|-|-|
+| **Rule ID** |AZFW00011|
+| **Category** |[Usage]|
+| **Severity** |Error|
+
+## Cause
+
+This rule is triggered when a function is binding to a non-iterable type for a  blob container path.
```

Comment:
This rule is triggered when a function is binding to a non-iterable type for a blob container path.